idle-rpg/game/utils/Event.js

```diff
@@ -89,6 +90,10 @@ class Event {
                   const eventLog = `Died to ${defender.name} in ${selectedPlayer.map.name}.`;
                   const otherPlayerLog = `Killed ${selectedPlayer.name} in ${selectedPlayer.map.name}.`;
 
+                  if (selectedPlayer.horse === 'Yes'){
+                    otherPlayerLog = `Killed ${selectedPlayer.name} in ${selectedPlayer.map.name}. ${selectedPlayer.name}'s horse also died in the fight.`
+                  }
```

Comment:
gotta update the other players eventlog aswell (possible use .concat() to add the horse string and have to change from const to let